docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -139,13 +140,13 @@ network_config:
   addresses:
   - ip_netmask: {{ external_ip }}/{{ external_cidr }}
   routes: {{ external_host_routes }}
-  members:
+  members: []
   - type: interface
     name: nic3
     primary: true
 ----
 
-.. Add the External Network to the `baremetal.yaml` file that is used by `metalsmith`:
+.. Add the External Network to the baremetal file, for example, `/home/stack/composable_roles/network/baremetal_deployment.yaml` that is used by `metalsmith`. Also 'network_config_update' should be enabled for network propogation to target nodes during `os-net-config`:
```

Comment:
```suggestion
.. Add the External Network to the bare metal file, for example, `/home/stack/composable_roles/network/baremetal_deployment.yaml` that is used by `metalsmith`:
+
[NOTE]
Ensure that  'network_config_update' is enabled for network propagation to the target nodes during `os-net-config`.
```
What do you mean by *during* `os-net-config`? Do we want to say something like "... is enabled for network propagation to the target nodes when `os-net-config` is triggered" ?

Reply:
>>What do you mean by during os-net-config? Do we want to say something like "... is enabled for network propagation >>to the target nodes when os-net-config is triggered" ?

@klgill yes you are right.